app/src/test/kotlin/fr/free/nrw/commons/upload/categories/UploadCategoriesFragmentUnitTests.kt

```diff
@@ -23,6 +23,7 @@ import fr.free.nrw.commons.OkHttpConnectionFactory
 import fr.free.nrw.commons.R
 import fr.free.nrw.commons.TestCommonsApplication
 import fr.free.nrw.commons.createTestClient
+import fr.free.nrw.commons.databinding.UploadCategoriesFragmentBinding
```

Comment:
Unused import optimized needed : 16 unused imports

Reply:
this is unrelated to the issue it seems but I'll do it.